app/javascript/dashboard/routes/dashboard/settings/inbox/channels/WhatsappEmbeddedSignup.vue

```diff
@@ -266,22 +279,7 @@ onBeforeUnmount(() => {
       </div>
 
       <div class="flex flex-col gap-2 mb-6">
-        <span class="text-sm text-n-slate-11">
-          {{ $t('INBOX_MGMT.ADD.WHATSAPP.EMBEDDED_SIGNUP.LEARN_MORE.TEXT') }}
-          {{ ' ' }}
-          <a
-            :href="
-              $t('INBOX_MGMT.ADD.WHATSAPP.EMBEDDED_SIGNUP.LEARN_MORE.LINK_URL')
-            "
-            target="_blank"
-            rel="noopener noreferrer"
-            class="underline text-primary"
-          >
-            {{
-              $t('INBOX_MGMT.ADD.WHATSAPP.EMBEDDED_SIGNUP.LEARN_MORE.LINK_TEXT')
-            }}
-          </a>
-        </span>
+        <span class="text-sm text-n-slate-11" v-html="learnMoreText" />
```

Comment:
This is not a good approach. Ref: [Vue i18n’s Component Interpolation](https://vue-i18n.intlify.dev/guide/advanced/component#component-interpolation)